insights/specs/__init__.py

```diff
@@ -430,6 +430,7 @@ class Specs(SpecSet):
     octavia_conf = RegistryPoint(filterable=True)
     odbc_ini = RegistryPoint(filterable=True)
     odbcinst_ini = RegistryPoint()
+    open_vm_tools_stat_raw_text_session = RegistryPoint(filterable=True)
```

Comment:
@xiangce please check the spec review regarding filtering.

Reply:
@bfahr - I removed the "fitlerable=True" per the discussion. Please re-check it. Thanks.